script/apps/Aries/mcml/pe_aries_window.lua

```diff
@@ -33,17 +33,20 @@ local mc_line = "Texture/Aries/Creator/Theme/GameCommonIcon_32bits.png;352 66 1
 
 function pe_aries_window.render_callback(mcmlNode, rootName, bindingContext, _parent, left, top, right, bottom, myLayout, css)
 	local mode = mcmlNode:GetString("mode") or "full"; -- full or lite
-
-	if(mode == "full")then
-		pe_aries_window.create_full(rootName, mcmlNode, bindingContext, _parent, left, top, right, bottom, myLayout, css);
-	elseif(mode == "thin" or mode == "mc")then
-		pe_aries_window.create_thin_mc(rootName, mcmlNode, bindingContext, _parent, left, top, right, bottom, myLayout, css);
-	elseif(mode == "center") then
-		pe_aries_window.create_center(rootName, mcmlNode, bindingContext, _parent, left, top, right, bottom, myLayout, css);
-	elseif(mode == "no_title") then
-		pe_aries_window.create_no_title(rootName, mcmlNode, bindingContext, _parent, left, top, right, bottom, myLayout, css);
-	else
-		pe_aries_window.create_lite(rootName, mcmlNode, bindingContext, _parent, left, top, right, bottom, myLayout, css);
+	local useDefaultStyle = GameLogic.GetFilters():apply_filters('AriesWindow.CustomStyle',rootName, mcmlNode, bindingContext, _parent, left, top, right, bottom, myLayout, css, mode);
```

Comment:
refactor this. 
add one more filter value like false.
useDefaultStyle = rootName？